src/main/java/de/tum/in/www1/artemis/domain/participation/ParticipantParticipation.java

```diff
@@ -0,0 +1,108 @@
+package de.tum.in.www1.artemis.domain.participation;
+
+import java.util.Optional;
+
+import javax.persistence.Column;
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+
+import org.hibernate.Hibernate;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.modeling.ModelingSubmission;
+
+@Entity
+@DiscriminatorValue(value = "PP")
+public abstract class ParticipantParticipation extends Participation {
```

Comment:
get rid of it

Reply:
I got rid of `ParticipantParticipation` and moved the `findLatest` methods up from `StudentParticipation` into `Participation`.